### DISCUSSION TITLE
Relation Manager Action Button Not showing

### DISCUSSION THREAD

**Discussion by MarJose123** [filamentphp/filament#8550] [Help]
I've disabled showing the Relation Manager in Edit page view by overriding the function, and wanted only to show the Relation Manager on resource record view.
```php
public function getRelationManagers(): array
{
    return [];
}
```

The relation manager action button is working properly in Edit Page, however in the record Page view the action button is not showing.

Is it a limitation under the filament development that it only allow as of now in edit page to render the action button?

This is what I only see in the resource record view page.
![image](https://github.com/filamentphp/filament/assets/18107626/98e052e7-88b8-47d9-9fb2-8a9e406a78b6)

**Comment by leandrocfe** [comment]
You can add this

```php
//YourRelationManager.php
public function isReadOnly(): bool
{
    return false;
    //return !Str::of($this->pageClass)->contains('MyViewPageName'); //only on the ViewPage
}
```

**Reply by rozaqabdul656** [reply]
thanks

**Reply by fadhilkgm** [reply]
any other solution? this is not working for me, i am using latest filamentphp version

**Reply by raves21** [reply]
why is this not default?

**Reply by MarJose123** [reply]
> any other solution? this is not working for me, i am using latest filamentphp version

Are you using v4 beta or v3? 

The solution above should work for v3.
